cookbook/entities/datagrid.rst

```diff
@@ -0,0 +1,201 @@
+                    taskPriority:
+                        data_name: priority.label
+                default:
+                    dueDate: DESC
+
+Each key under ``sorters.columns`` refers to one of the displayed columns. The ``data_name`` option
+is the term that will be used as the ``order by`` term in the Doctrine query.
+
+Data Filters
+~~~~~~~~~~~~
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/AppBundle/Resources/config/datagrid.yml
+    datagrid:
+        app-tasks-grid:
+            # ...
+            filters:
+                columns:
+                    id:
+                        type: number
+                        data_name: task.id
+                    subject:
+                        type: string
+                        data_name: task.subject
+                    description:
+                        type: string
+                        data_name: task.description
+                    dueDate:
+                        type: datetime
+                        data_name: task.dueDate
+                    taskPriority:
+                        type: string
+                        data_name: priority.label
+
+.. _cookbook-entities-grid-controller:
+
+Create the Controller and View
+------------------------------
+
+To make your datagrid accessible you need to create a controller that can be visited by the user
+which will serve a view that renders the configured datagrid:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Controller/TaskController.php
+    namespace AppBundle\Controller;
+
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+    /**
+     * @Route("/task")
+     */
+    class TaskController extends Controller
+    {
+        /**
+         * @Route("/", name="app_task_index")
+         * @Template()
+         */
+        public function indexAction()
+        {
+            return array();
+        }
+    }
+
+The view can be very simple if you extend the ``OroUIBundle:actions:index.html.twig`` template:
+
+.. code-block:: html+jinja
+    :linenos:
+
+    {# src/AppBundle/Resources/views/Task/index.html.twig #}
+    {% extends 'OroUIBundle:actions:index.html.twig' %}
+
+    {% set gridName = 'app-tasks-grid' %}
+    {% set pageTitle = 'Task' %}
+
+You simply need to configure the name of your datagrid and the title you wish to be displayed.
+Everything else is handled by the base template from the OroUIBundle.
+
+.. _cookbook-entities-grid-navigation:
+
+Link to the Action
+------------------
+
+At last, you need to make the action accessible by creating a menu item:
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/AppBundle/Resources/config/navigation.yml
+    oro_menu_config:
+        items:
+            task_list:
+                label: Tasks
+                route: app_task_index
+        tree:
+            application_menu:
```

Comment:
Please add the note that `application_menu` in this example is the menu name. So here could be another name.